doc/design/upgrade.md

```diff
@@ -26,7 +45,9 @@ There will be impact on container interaction while the appliance is down:
 - no attach ability
 - network outage for NAT based port forwarding
 
-All of those facets should resume normal operation after upgrade is complete
+All of those facets should resume normal operation after upgrade is complete. 
```

Comment:
The port mapping info is not stored in the kv store; instead at personality bring up the port mapping information is retrieved from the port layer as part of each container's configuration (during cache "sync up"). The port mapping info is stored as part of the containers' guestinfo.